'TextEncoder' refers to a value, but is being used as a type here

Getting the following error when compiling in Typescript:

node_modules/@cloudamqp/amqp-client/types/amqp-base-client.d.ts:27:27 - error TS2749: 'TextEncoder' refers to a value, but is being used as a type here. Did you mean 'typeof TextEncoder'?

27     readonly textEncoder: TextEncoder;

This is version - 3.1.1

If I comment out 
line 27 in 
node_modules/@cloudamqp/amqp-client/types/amqp-base-client.d.ts

everything works.

--- a/src/amqp-base-client.ts
+++ b/src/amqp-base-client.ts
@@ -28,7 +28,7 @@ export abstract class AMQPBaseClient {
   onerror: (error: AMQPError) => void
   logger: Logger | null | undefined = console
   /** Used for string -> arraybuffer when publishing */
-  readonly textEncoder = new TextEncoder()
+  readonly textEncoder: InstanceType<typeof TextEncoder> = new TextEncoder()
   // Buffer pool for publishes, let multiple microtasks publish at the same time but save on allocations
   readonly bufferPool: AMQPView[] = []
 

